src/vellum/workflows/constants.py

```diff
@@ -58,3 +58,7 @@ class APIRequestMethod(Enum):
 class AuthorizationType(Enum):
     BEARER_TOKEN = "BEARER_TOKEN"
     API_KEY = "API_KEY"
+
+
+class VellumIntegrationProviderType(Enum):
+    COMPOSIO = "COMPOSIO"
```

Comment:
[oos] I would expect us to be able to replace this with the client sdk imports once we publish those API's